packages/examples/cvat/exchange-oracle/src/services/cloud/__init__.py

```diff
@@ -1,4 +1,3 @@
 from src.services.cloud.client import StorageClient
-from src.services.cloud.s3 import S3Client
```

Comment:
If you decided to remove S3Client import, please update tests ([1](https://github.dev/Marishka17/human-protocol/blob/b0568928c2fa55b25e0fe14951ed95c816862e03/packages/examples/cvat/exchange-oracle/tests/integration/cron/state_trackers/test_retrieve_annotations.py#L133), [2](https://github.dev/Marishka17/human-protocol/blob/2d57d96af21b829b39b074d807025c69f628f467/packages/examples/cvat/exchange-oracle/tests/integration/cron/state_trackers/test_retrieve_annotations.py#L254)) or revert back

Reply:
Seems like these tests do not depend on this class anymore.